snafu/dns_perf_wrapper/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.centos.org/centos:8
+
+COPY snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+RUN dnf install -y --nodocs python3 python3-pip epel-release && \
+    dnf install -y openssl-devel ck-devel yum-plugin-copr && \
+    dnf copr enable @dnsoarc/dnsperf -y && \
+    dnf install dnsperf -y && \
+    dnf clean all && rm -rf /var/cache/yum
+RUN dnf install -y --nodocs redis --enablerepo=centos8-appstream && dnf clean all
```

Comment:
This should be moved above the `install / clean ` to reduce redundancy.

Reply:
Fixed it, thanks.